lib/rubocop/cop/rails/active_record_aliases.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks that ActiveRecord aliases are not used. The direct method names
+      # are more clear and easier to read.
+      #
+      # @example
+      #   #bad
+      #   Book.update_attributes!(author: 'Alice')
+      #
+      #   #good
+      #   Book.update!(author: 'Alice')
+      class ActiveRecordAliases < Cop
+        MSG = 'Use `%<prefer>s` instead of `%<current>s`.'.freeze
+
+        ALIASES = {
+          update_attributes: :update,
+          update_attributes!: :update!
+        }.freeze
+
+        def on_send(node)
+          ALIASES.each do |bad, good|
+            next unless node.method?(bad)
+
+            add_offense(node,
+                        message: format(MSG, prefer: good, current: bad),
+                        location: :selector,
+                        severity: :warning)
+            break
+          end
+        end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            corrector.replace(
+              node.source_range,
+              node.source.gsub('update_attributes', 'update')
+            )
+          end
```

Comment:
Can you use `node.loc.selector` instead of `node.source_range`, and `ALIASES[node.method_name].to_s` for replacement?
Current implementation breaks code if a receiver has "update_attributes".

```ruby
# before
update_attributes_foobar.update_attributes!(foo: bar)
# after
update_foobar.update!(foo: bar)
```

Reply:
Changed to the suggested code and added a test case.